cg/constants/constants.py

```diff
@@ -162,10 +162,12 @@ class FileFormat(StrEnum):
 
 
 class GenomeVersion(StrEnum):
+    GRCh37: str = "GRCh37"
+    GRCh38: str = "GRCh38"
```

Comment:
Any chance we can give these descriptive names instead? If there is such a name 🥹 Like `HUMAN_37` or something

Reply:
The problem is that both terms refer to the same human genome assembly, and we also have more complex names like `T2T_CHM13` for a specific cell line. I would keep the technical notation to keep the names short and consistent with other versions